src/objects.rs

```diff
@@ -1277,9 +1288,81 @@ pub struct ODRule {
     pub network_id: Option<String>,
     pub physical_mode_id: Option<String>,
 }
+impl_id!(ODRuleV1);
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[cfg(feature = "fare-v2")]
+pub struct Ticket {
+    pub id: String,
+    pub name: String,
+    pub comment: String,
+}
+#[cfg(feature = "fare-v2")]
 impl_id!(Ticket);
-impl_id!(ODRule);
+
+#[cfg(feature = "fare-v2")]
+impl GetObjectType for Ticket {
```

Comment:
why do you need that?

Reply:
In the reference, it is mentioned the [following sentence](https://github.com/prhod/navitia/blob/doc_new_fares/documentation/ntfs/ntfs_fare_extension.md#other-modifications)

> The `object_properties.txt` and `object_codes.txt` files are extended to enable the reference to a ticket_id.

The serialization of these two files are using the trait `GetObjectType` if I understand correctly so I thought I would do that right now. But I'm not 100% sure I understand fully what I'm doing :rofl:, can you confirm it does what I think it does or confirm it's useless?